### DISCUSSION TITLE
Column 'active' in field list is ambiguous

### DISCUSSION THREAD

**Discussion by frevab** [PrestaShop/PrestaShop#31160] [Q&A]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

I experience major problems with my webshop since I moved host. 

Login to BO was ok but then I changed shop maintanance setting to put shop active but then I lost access to both BO and FO.
The error is above. 
However the database user and database name do not exist anymore and never have on the new host. what is shown in this error is not what is shown in the app>config> parameters.php settings.
How is this possible?  where are these coming from.  I have pinged my domain to make sure it does not link to the old host and it shows to the IP of my new host.
The database credentials were correct and stored in an old paramaters.php file which was part of the back-up files I used to import to the new host. I copied the old paramater.php file and renamed it to parameter-kopie.php and amended the parameter.php to the new database credentials. This was working correctly and I could login in BO until I moved the slider to put shop from maintannce to live.
DAtabase on my new host I can access without problems so is not offline.

### Expected behavior

not to have this error

### Steps to reproduce

1. login in BO
2. change maintanance setting to shop enable

### PrestaShop version(s) where the bug happened

1.7.2.4

### PHP version(s) where the bug happened

7.1.33

**Comment by matks** [comment]
@frevab I converted your bug report into a discussion because I know for sure it is not a bug inside the software it is a problem in how you managed the change

If your prestashop continues using old credentials it is because it fetches them from an old location.

> The database credentials were correct and stored in an old paramaters.php file which was part of the back-up files I used to import to the new host. I copied the old paramater.php file and renamed it to parameter-kopie.php and amended the parameter.php to the new database credentials. This was working correctly

This was the correct thing to do 👍 

> until I moved the slider to put shop from maintannce to live.
DAtabase on my new host I can access without problems so is not offline.

I am afraid I dont understand how moving a slider could trigger that :( I suggest you scan all your shop files to look for old credentials they might have been copied-pasted in a file read by prestashop but we cant help you as this situation is only related to the changes you did
